### DISCUSSION TITLE
unfix los

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#3446] [Suggestions]
nothing else really needs to be said here
unfix los in wsg 
you may say its not blizzlike
neither is transmog on 3.3.5, mats exchange for tbc mats, tuning on vanilla raids, dks not being available till 71, Soul shard vendor, marks for wsg, av, ab. 
Sure it maybe a bug and may be hard to roll back the change but it makes the game way better to play. Changing LoS was indirectly a huge buff for casters in a bracket where they already were the best

**Comment by balleny** [comment]
Not sure if "hanging LoS was indirectly a huge buff for casters in a bracket where they already were the best" is really the truth.
Hunters are very strong, but mostly based on their pets / missing resilience.
Yes, melees miss some abilities / gap closers but casters have such bad gear available, miss even more key abilities etc.

**Reply by Vitusin** [reply]
GMs already said its not possible bcs it will break many other things in game. So no point of asking ever again. Los not happening

**Comment by Nyeriah** [comment]
Hello,

Thanks for your suggestion.

As aforementioned, this is how the game was designed and is still the behavior we see in the official servers up to this day.

We understand there's a widespread sentiment of disappointment with this change and we plan to further address it in the future as we analyze the resources and alternatives we have at our disposal.

Unfortunately, I haven't got further information to provide about this but certain matters are still up to deliberation. 

Regards.

**Comment by Nyeriah** [comment]
This suggestion has been accepted and as such the LoS behavior in battlegrounds has been reverted to how it was before. Locking.
